product_docs/docs/migration_toolkit/55/installing/macos.mdx

```diff
@@ -1,12 +1,12 @@
 ---
-title: "Installing on Mac OS X"
-navTitle: "On Mac OS X"
+title: "Installing on macOS"
+navTitle: "On macOS"
 redirects:
     - /migration_toolkit/latest/05_installing_mtk/install_on_mac/
     - /migration_toolkit/latest/installing/install_on_mac/
 ---
 
-EDB provides a graphical interactive installer for MacOS. You can access it two ways:
+EDB provides a graphical interactive installer for macOS. You can access it two ways:
```

Comment:
We are no longer building macOS installers. References to macOS should be removed.